caiman/source_extraction/cnmf/spatial.py

```diff
@@ -12,10 +12,14 @@
 from scipy.sparse import coo_matrix, csc_matrix, csr_matrix
 from scipy.sparse import spdiags
 from scipy.linalg import eig
-from scipy.ndimage import label, binary_dilation
-from scipy.ndimage.filters import median_filter
-from scipy.ndimage.morphology import binary_closing
-from scipy.ndimage.morphology import generate_binary_structure, iterate_structure
+from scipy.ndimage import (
```

Comment:
Did scipy flatten their imports?

Reply:
yup, they seem to have gotten rid of everything under `ndimage`: